docs/user-guide/diffing_known_types.txt

```diff
@@ -0,0 +1,205 @@
+core/v1/AWSElasticBlockStoreVolumeSource
```

Comment:
I'm unsure whether this file should live within docs - maybe it's better to place it in `hack` or even in `util/argo/normalizers` and point from the docs to the GH repository?

Reply:
moved